.gitignore

```diff
@@ -134,3 +134,7 @@ _start
 _temp_*/**/*
 
 src/.idea/
+
+frontend/\.idea/
```

Comment:
Do you need the backslash here?

Reply:
these were added automagically by my editor.. I will correct